### PR TITLE
Adds a new spell, Repulse.

### PR DESCRIPTION
**Pull request by Boggart**
Repulse has a cooldown of 40 seconds, 15 with max upgrade.

It knocks down (short, weaken(2)) and throws away everything within a 5 tile range (thowforce depends on the distance from the caster, minimum of three, max of 5). If you're standing on someone when you use it they take damage and are stunned for longer. (weaken(5))

It's mostly intended for throwing back pursuers and making them drop their guns, but you can also launch grenades back at the thrower, launch soap at pursuers and shoot hoops with it.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
It should be 10-7 with max upgrade. Other then that, :+1:

**Comment by pudl** [comment]
SHOOTING HOOPS WILL ALWAYS BE REMEMBERED :+1:

**Comment by Boggart** [comment]
I'm not sure buffing the upgrade cooldown reduction to 10 seconds or less at max is a good idea.

**Comment by Jordie0608** [comment]
Changelog

**Comment by Jalleo** [comment]
@Incoming5643 does this conflict with your PR?

**Comment by Incoming5643** [comment]
Instant summons already got merged, so if this pull is still automergable everything should be fine.

File: code/datums/spells/wizard.dm [hunk 1]
```diff
@@ -257,4 +257,48 @@
...
+			if(istype(AM, /mob/living))
+				var/mob/living/M = AM
+				M.Weaken(5)
+				M.adjustBruteLoss(5)
+				M << "You're slammed into the floor by a mystical force!"
+		else
+			if(istype(AM, /mob/living))
+				var/mob/living/M = AM
+				M.Weaken(2)
+				M << "You're thrown back by a mystical force!"
```
Comment (tkdrg) [inline code/datums/spells/wizard.dm]: Could you add a `<span class='userdanger'>` to these two messages, please?

**Comment by tkdrg** [comment]
Looks fine to me apart from that minor nitpick. :+1:

**Comment by Boggart** [comment]
Span classes added.

**Pull request merged**